src/containers/StructurePage/StructureContainer.tsx

```diff
@@ -239,6 +239,8 @@ export const StructureContainer = ({ match, location, history }: Props) => {
     setShowFavorites(!showFavorites);
   };
 
+  const isTaxonomyAdmin = userAccess?.includes(TAXONOMY_ADMIN_SCOPE) ?? false;
```

Comment:
Har `?? false` betydning eller kan det fjernes?

Reply:
Ingen annen betydning enn for typen. Trengs egentlig ikke her. Kan fjernes